.github/workflows/test.yaml

```diff
@@ -0,0 +1,52 @@
+name: test
+
+on:
+  pull_request:
+    branches: ['*']
+
+jobs:
+  test:
+
```

Comment:
are the blank lines necessary?

Reply:
They're not necessary, but it seems to be pattern people follow: https://docs.github.com/en/free-pro-team@latest/actions/quickstart and https://github.com/uber-go/zap/commit/02a233f7b9041b29460a7372f788b324dbd64c4e.

Could remove but it didn't seem horrible?